### ISSUE TITLE
Content Strategy: Weather event recaps/reports

### ISSUE THREAD

**Issue by colinmurphy01** [weather-gov/weather.gov#1378]
Based on [initial discovery](https://docs.google.com/document/d/1JD0lLGEEPUtgBsgqpFsIEOTn1ec9EAdga1T4cs127MM/edit#heading=h.ik527p444oic)

- Answer open questions
- Scope what must, should, could, won't be on the site
- Discuss as a group
- Determine if need to `prototype` or ready to 'design` or `build`

High level background: 
- Conversation with forecaster during PA indicated this content was primarily for people (gen pop) impacted by the storm to report to insurance, and EMs at any scale to coordinate response. 
- Conversation with POs indicated that insurers do not use this content at all, and that primary users are generally media. 
- Conversation with area partners in OHX included one vague remark about using the storm track to distribute post-event recovery resources. 

Supporting data: 
Pulled some google analytics around event recap pages. The numbers are in the qualifying entries on the "event recaps" tab in this [spreadsheet](https://docs.google.com/spreadsheets/d/1F1WHRVOIuOch91G4DqQsbBqGH6yx1uiXJXRMcucIzuM/edit?usp=sharing).
The takeaways are essentially that some of these pages receive very little traffic, but others receive very high (in the tens of thousands of pageviews). It is a generally true pattern that the bulk of the traffic hits in the first month after the event, and then slows down dramatically. However, patterns could be misleading because this sample is limited to event pages I could easily find that are for events within the past year.

**Comment by colinmurphy01** [comment]
@kmranjo hello from this ticket.

My thinking on these traffic analytics is that despite them being lower than most pages:
- the area they cover and population they serve is smaller than the national-level pages 
- they often get repeat visitors

This leads me to conclude that as a whole this is a medium to high value piece of content for people. Do you agree?

**Comment by kmranjo** [comment]
I agree on the point that they serve an audience thats not at the national scale. The others I'm less sure about. 
I don't have strong evidence that they get repeat visitors - there were a couple pages that stood out because the number of pageviews was so much higher than the number of users, indicating several return visits. But those stood out because they were not the norm. Most often (again, just in this limited sample) the number of pageviews and visitors was pretty close to even, indicating just one visit. 
I'm hesitant to draw any conclusions about the value of this content just yet. All we have on it so far is one conversation with a forecaster, one with a PO, and some analytics on a specific sample of pages. We haven't spoken at all to anyone who uses the content yet, and of the two conversations we did have about it, we got entirely different information about the primary audience and purpose. (I'll put some notes from that in the ticket above).
I'd recommend talking to more people about what this is and what its used for. Unless NWS HQ determines that none of the possible use cases we've heard should be priority, then that seems worth the time we'd spend.
